_includes/step-nav.html

```diff
@@ -45,28 +45,30 @@ <h3>Part 2 - The Browser</h3>
   {% endfor %}
   </ul>
 
-  <!-- <h3>Challenges</h3>
+  <h3>Level 3</h3>
   <ul>
-  {% for challenge in site.challenges %}
-      <li class='{% if page.cnumber == challenge.cnumber %}active{% endif %}'>
-        <a href="{{ challenge.url | relative_url }}">
-          {% if challenge.cnumber != "start" %}{{challenge.cnumber}} - {% endif %}{{ challenge.title | escape }}
+  {% for step in steps %}
+  {% if step.level == 3 %}
+      <li class='{% if page.number == step.number %}active{% endif %}'>
+        <a href="{{ step.url | relative_url }}">
+          {% if step.number != "start" %}{{step.number}} - {% endif %}{{ step.title | escape }}
         </a>
       </li>
+      {% endif %}
   {% endfor %}
-  </ul> -->
+  </ul>
 
-  <ul class="main">
+  <!-- <ul class="main"> -->
     <!-- <li class="{% if page.url == '/troubleshooting/' }} %}active{% endif %}"><a href="{{ '/troubleshooting/' | relative_url }}">Troubleshooting</a></li> -->
-    <li class="{% if page.url == '/journey-continues/' }} %}active{% endif %}"><a href="{{ '/journey-continues/' | relative_url }}">The Journey Continues</a></li>
-  </ul>
+    <!-- <li class="{% if page.url == '/journey-continues/' }} %}active{% endif %}"><a href="{{ '/journey-continues/' | relative_url }}">The Journey Continues</a></li> -->
+  <!-- </ul> -->
```

Comment:
Would be good to leave this in, so we can update with additional post-workshop JS resources.

Reply:
This was mostly left out (temporarily) while I tried to figure out how to get Jekyll to adjust the page height if the content is smaller than the navigation - currently this part of the nav won't even appear on some pages. I'll add it back in for now and keep trying to figure it out.